src/pages/Home/components/Days/DaysStyles.ts

```diff
@@ -0,0 +1,86 @@
+import styled from '@emotion/styled'
+export const DaysStyles = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding:20px;
+  width:100%;
+  background: #fff;
+  box-shadow: 2px 5px 25px -3px rgba(100,100,100,0.25);
+  border-radius: 0 0 20px 20px;
+`
+export const CardStyle = styled.div`
+  display: flex;
+  flex-direction: column;
+  background: rgba(71,147,255,0.2);
+  border-radius: 10px;
+  padding: 10px;
+  width: 149px;
+  transition: 0.3s;
+  cursor:pointer;
+  &:hover{
+    box-shadow: 0 3px 20px rgba(147,144,144,0.28);
+  }
+`
+export const DayStyle = styled.div`
+  margin-bottom: 7px;
+  font-size: 18px;
+`
+export const DayInfo = styled.div`
+  font-size: 14px;
+  margin-bottom: 12px;
+  color: #939cd0;
+`
+export const Img = styled.div`
+  margin-bottom: 12px;
+`
+export const DayTemp = styled.div`
+  font-size: 18px;
+`
+export const NightTemp = styled.div`
+  font-size: 13px;
+  color: #939cd0 ;
+  margin-bottom: 6px;
+`
+export const Info = styled.div`
+  font-size: 13px;
+  color:#939cd0;
+`
+export const TabsStyle = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 7px;
+`
+export const TabsWrapper = styled.div`
+  display: flex;
+`
+export const TabStyle = styled.div`
```

Comment:
You need create two different files, for example: buttons.ts and cards.ts. This tab is a button as each other. You have to create less Button (styled) as you can. Follow this example
```suggestion
const buttonProperties = new Set([
  'primary',
  'tab',
  'active'
]);

export const Button = styled('div', {
   shouldForwardProp: filterProperties(buttonProperties)
})`
  display: inline-block;
  text-align: center;
  vertical-align: middle;
  user-select: none;
  background-color: transparent;
  border: 1px solid transparent;
  transition: all 0.25s;
  cursor: pointer;

  ${props => `
    border-radius: ${props.theme.button.borderRadius};
    font-weight: ${props.theme.button.fontWeight};
    line-height: ${props.theme.button.lineHeight};
    font-size: ${props.theme.button.fontSize};
    padding-top: ${props.theme.button.padding.top};
    padding-bottom: ${props.theme.button.padding.bottom};
    padding-left: ${props.theme.button.padding.left};
    padding-right: ${props.theme.button.padding.right};
  `}
  
  ${props => props.tab && `
    margin: ${props.theme.button.colors.tab.margin};
    padding: ${props.theme.button.colors.tab.padding};
    color: ${props.theme.button.colors.tab.color};
    font-size: ${props.theme.button.colors.tab.fontSize};
    text-transform: ${props.theme.button.colors.tab.textTransform};
    font-weight: ${props.theme.button.colors.tab.fontWeight};
    letter-spacing: ${props.theme.button.colors.tab.letterSpacing};
    line-height: ${props.theme.button.colors.tab.lineHeight};
    border-bottom: ${props.theme.button.colors.tab.borderBottom};

    ${props.active && `
      border-bottom: ${props.theme.button.colors.tab.active.borderBottom};
      color: ${props.theme.button.colors.tab.active.color};
    `}
  `}
`
```